samples/bluetooth/peripheral_status/README.rst

```diff
@@ -46,36 +46,36 @@ RGB LED:
    The RGB LED channels are used independently to display the following information:
 
    * Red - If the main loop is running (that is, the device is advertising).
-     The LED blinks with a period of 2 seconds, duty cycle of 50%.
+     The LED blinks with a period of two seconds, duty cycle of 50%.
```

Comment:
This can also be made more in line with the LED description used in the other samples.